data/org.freedesktop.portal.GameMode.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0"?>
+<!--
+ Copyright (C) 2019 Red Hat, Inc.
+ This library is free software; you can redistribute it and/or
+ modify it under the terms of the GNU Lesser General Public
+ License as published by the Free Software Foundation; either
+ version 2 of the License, or (at your option) any later version.
+ This library is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ Lesser General Public License for more details.
+ You should have received a copy of the GNU Lesser General Public
+ License along with this library. If not, see <http://www.gnu.org/licenses/>.
+ Author: Christian J. Kellner <christian@kellner.me>
+-->
+
+<node name="/" xmlns:doc="http://www.freedesktop.org/dbus/1.0/doc.dtd">
+    <!--
+      org.freedesktop.portal.GameMode:
+      @short_description: Portal for accessing GameMode
+
+      Interface for accessing GameMode from within the sandbox. It
+      is analogous to the com.feralinteractive.GameMode interface
+      and will proxy request there, but with permission checking and
+      pid mapping (from the sandbox' pid namespace to the host one)
+      if necessary.
+
+      This documentation describes version 1 of this interface.
+    -->
+    <interface name="org.freedesktop.portal.GameMode">
+      <!--
+          QueryStatus:
+          @pid: Process id to query the GameMode status of
+          @result: The GameMode status for @pid
+
```

Comment:
Should probably say something about pid namespaces here ?

Reply:
I added some small explanation to all the methods and a more detailed introduction to the general doc section